GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeViewModel.swift

```diff
@@ -128,13 +128,14 @@ private extension SecureConversations.WelcomeViewModel {
         _ = environment.sendSecureMessagePayload(
             payload,
             queueIds
-        ) { [weak self, alertConfiguration = environment.alertConfiguration] result in
+        ) { [weak self, alertConfiguration = environment.alertConfiguration, environment] result in
             self?.sendMessageRequestState = .waiting
 
             switch result {
             case .success:
                 self?.delegate?(.confirmationScreenRequested)
             case .failure:
+                environment.log.prefixed(Self.self).info("Show Unexpected error Dialog")
```

Comment:
would not it be beneficial to log specific error here?

Reply:
https://github.com/salemove/ios-sdk-widgets/pull/852#issuecomment-1854455410